relayer/chain/ethereum/listener.go

```diff
@@ -117,23 +140,64 @@ func (li *Listener) pollEventsAndHeaders(
 			}
 
 			finalizedBlockNumber := gethheader.Number.Uint64() - descendantsUntilFinal
-			var events []*outbound.ContractMessage
-
+			var channelEvents []*outbound.ContractMessage
 			for _, channelContract := range li.contracts {
-				channelEvents, err := li.queryEvents(ctx, channelContract, finalizedBlockNumber, &finalizedBlockNumber)
+				contractEvents, err := li.queryChannelEvents(ctx, channelContract, finalizedBlockNumber, &finalizedBlockNumber)
 				if err != nil {
 					li.log.WithError(err).Error("Failure fetching event logs")
 				}
 
-				events = append(events, channelEvents...)
+				channelEvents = append(channelEvents, contractEvents...)
+			}
+			li.forwardChannelEvents(ctx, hcs, channelEvents)
+
+			// Query PolkadotRelayChainBridge contract's InitialVerificationSuccessful events
+			blockNumber := gethheader.Number.Uint64()
+			var events []*polkadotrelaychainbridge.ContractInitialVerificationSuccessful
```

Comment:
events -> relayChainBridgeEvents

Reply:
Updated in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/bc765754f8d0e0df068a2b39d0e6bf1225b6eb18